src/styles/blocks/header.scss

```diff
@@ -0,0 +1,70 @@
+body {
+  margin: 0;
+  box-sizing: border-box;
+  font-family: "Roboto", serif;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 40px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__img-logo {
+    margin: 10px 0 0 20px;
```

Comment:
try to set  object-fit: none; instead of margins
Make sure to check if it works